css/bundle/os-mt-ult.css

```diff
@@ -472,8 +472,8 @@ label.mt-feedback-consent-label {
 
     .elm-content-footer-ult-fixed {
         position: fixed !important;
-        top: 0;
-        right: 6%;
+        top: 70px;
```

Comment:
On my screen it jumps less with 60px. We need to check if it's resolution dependent.